### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by fisker**
## Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To format a file in-place, use `--write`. (Note: This overwrites your files!)
 In practice, this may look something like:
 
 ```bash
-prettier --write .
+prettier . --write
```
Comment (fisker) [inline docs/cli.md:19]: Am I the only one? I always type patterns first.

File: docs/ignore.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ To exclude files from formatting, create a `.prettierignore` file in the root of
 
 Example:
 
-```
```
Comment (thorn0) [inline docs/ignore.md:16]: What's the point of marking it as `text`?

**Review by thorn0 — APPROVED** [review]

File: docs/ignore.md (hunk 2 above)
Comment (fisker) [reply docs/ignore.md:16]: To satisfy https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md040---fenced-code-blocks-should-have-a-language-specified, I can remove it.

Comment (thorn0) [reply docs/ignore.md:16]: I just was curious.

Comment (fisker) [reply docs/ignore.md:16]: I'll keep it then. I'll need update docs in #13530

**Pull request merged**